bin/start_qc_docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+image="$1"
+if [ -z "$image" ]; then
+  echo "You must supply the image name as the first argument to this script!"
+  exit 1
+fi
+
+local_config_dir="$(dirname "$(realpath "$0")")/../config"
```

Comment:
Does this require reworking of the config file?

🆕 👨 ❓

Reply:
YES. added.